udata/core/dataset/tasks.py

```diff
@@ -53,3 +61,64 @@ def crawl_resources(self):
     }))
     log.info('URLs checked with group hash {hash}'.format(
         hash=response.json()['group-hash']))
+
+
+@job('send-frequency-reminder')
+def send_frequency_reminder(self, days_left_to_producer=7):
+    del UPDATE_FREQUENCIES['unknown']
+    del UPDATE_FREQUENCIES['realtime']
+    del UPDATE_FREQUENCIES['punctual']
```

Comment:
I'm open to better suggestions, iterating over the dict just to remove these 3 items looks more costly to me.